alivee/alivee/settings.py

```diff
@@ -37,6 +37,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'place_rate_engine',
```

Comment:
let's update the INSTALLED_APPS in `dev_settings.py` as well for the future sake.